gp-includes/template.php

```diff
@@ -582,3 +582,60 @@ function gp_entry_actions( $seperator = ' &bull; ' ) {
 	<a href="#" class="gtranslate" tabindex="-1"><?php _e( 'Translation from Google', 'glotpress' ); ?></a>
 	*/
 }
+
+/**
+ * Returns the warning class for the translation row.
+ *
+ * @param Translation_Entry $t translation entry for the row.
+ *
+ * @since 2.2.0
+ *
+ * @return string
+ */
+function gp_get_translation_row_warning_class( $t ) {
+	$warning_class = $t->warnings? 'has-warnings' : 'no-warnings';
+	return $warning_class;
+}
```

Comment:
I don't see the need for this extra function, this can be part of `gp_get_translation_row_classes()`.